docs/readme-sync/sdk-reference-guides/csharp-sdk/040 - optimizelyconfig-csharp.md

```diff
@@ -10,7 +10,7 @@ updatedAt: "2020-01-28T21:53:11.290Z"
   "title": "Overview"
```

Comment:
Change looks good, but md file as a whole doesn't seem to be rendering properly. Could you please take a look?